examples/inline_bot/src/main.rs

```diff
@@ -0,0 +1,73 @@
+use teloxide::{Bot, payloads::AnswerInlineQuery, prelude::*, types::{InlineQueryResult, InlineQueryResultArticle, InputMessageContent, InputMessageContentText}};
+use tokio_stream::wrappers::UnboundedReceiverStream;
+
+#[tokio::main]
+async fn main() {
+    run().await;
+}
+
+async fn run() {
+    let bot = Bot::from_env();
+    // Create a new dispatcher to handle incoming messages
+    let dp = Dispatcher::new(bot);
+    dp.inline_queries_handler(|rx: DispatcherHandlerRx<Bot, InlineQuery>| {
+        UnboundedReceiverStream::new(rx).for_each_concurrent(None, |msg| async move {
```

Comment:
```suggestion
        UnboundedReceiverStream::new(rx).for_each_concurrent(None, |query| async move {
```

That's not a message, we are handling inline queries here.